_posts/2019-10-11-stripe-connect-in-rails-part-2.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "Implementing Stripe Connect in Rails: Part 2"
+date: 2019-10-11 14:00:00
+categories: ["rails", "stripe"]
+author: "luciano"
+---
+
+Some time ago we wrote [an article](https://www.ombulabs.com/blog/rails/stripe/stripe-connect-part-1.html) to explain how to implement [Stripe Connect](https://stripe.com/connect) in a [Rails](https://rubyonrails.org) application. That article covered mainly the connection part between Rails and Stripe. This one will cover the fun part, which is making transactions, charges, refunds, and more.
+
+If you haven't check the [first part](https://www.ombulabs.com/blog/rails/stripe/stripe-connect-part-1.html) yet I recommend you to do it since this article will continue from where that one ended.
```

Comment:
```suggestion
If you haven't checked out the [first part](https://www.ombulabs.com/blog/rails/stripe/stripe-connect-part-1.html) yet I recommend you do so since this article will continue from where that one ended.
```